cmake/python.cmake

```
@@ -1,7 +1,13 @@
 # the CMake variable PYTHON_INSTALL_DIR has the same value as the Python function catkin.builder.get_python_install_dir()
 
 set(PYTHON_VERSION "$ENV{ROS_PYTHON_VERSION}" CACHE STRING "Specify specific Python version to use ('major.minor' or 'major')")
-find_package(PythonInterp ${PYTHON_VERSION} REQUIRED)
+find_package(Python ${PYTHON_VERSION} REQUIRED)
```

According to the [docs](https://cmake.org/cmake/help/v3.12/module/FindPython.html#findpython), you should specify both components, `Interpreter` and `Development`:
```suggestion
find_package(Python ${PYTHON_VERSION} REQUIRED COMPONENTS Interpreter Development)
```